ludwig/models/model.py

```diff
@@ -447,9 +447,9 @@ def train(
         train_writer = None
         if is_on_master():
             if not skip_save_log:
+                # todo tf2: completed fix
                 train_writer = tf2.summary.create_file_writer(
-                    os.path.join(save_path, 'log', 'train'),
-                    # session.graph  # todo tf2: fix this
```

Comment:
fine for now, we have to check how does the filewriter works in TF2. Before if you specified the graph it would save the summary metrics in the graph (look for instance at the measure/losses of binary features, they all have their summaries) and the graph itself so that it would be inspected in the graph tab of tensorboard. We need to find how to reimplement this functionality in TF2.